README.md

```diff
@@ -1,3 +1,5 @@
+[![CI](https://github.com/atlp-rwanda/phantom-be-codebandits/actions/workflows/main.yml/badge.svg)](https://github.com/atlp-rwanda/phantom-be-codebandits/actions/workflows/main.yml) [![Maintainability](https://api.codeclimate.com/v1/badges/0a8e73d7e498e308d3db/maintainability)](https://codeclimate.com/github/atlp-rwanda/phantom-be-codebandits/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/0a8e73d7e498e308d3db/test_coverage)](https://codeclimate.com/github/atlp-rwanda/phantom-be-codebandits/test_coverage) [![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/phantom-be-codebandits/badge.svg?branch=ch-setup-ci-and-qa-pipeline-cp-18)](https://coveralls.io/github/atlp-rwanda/phantom-be-codebandits?branch=ch-setup-ci-and-qa-pipeline-cp-18)
```

Comment:
Change the branch in coveralls badge to be `develop` since this one will be merged , the badge will reflect the new build. Do that change in **2 places**